docs/conf.py

```diff
@@ -1,186 +1,122 @@
-man_pages = [('index', project.lower(), project + u' Documentation',
-              [author], 1)]
+rst_epilog = """
+.. _Astropy: http://astropy.org
+"""
+# -- Options for HTML output ----------------------------------------------
 
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
 
-# -- Options for the edit_on_github extension ---------------------------------
+import sphinx_rtd_theme
+html_theme = 'sphinx_rtd_theme'
+html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
-if eval(setup_cfg.get('edit_on_github')):
-    extensions += ['astropy_helpers.sphinx.ext.edit_on_github']
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+# html_theme_options = {}
 
-    versionmod = __import__(setup_cfg['package_name'] + '.version')
-    edit_on_github_project = setup_cfg['github_project']
-    if versionmod.version.release:
-        edit_on_github_branch = "v" + versionmod.version.version
-    else:
-        edit_on_github_branch = "master"
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+#html_static_path = ['_static']
 
-    edit_on_github_source_root = ""
-    edit_on_github_doc_root = "docs"
+html_sidebars = {
+    '**': ['localtoc.html'],
+    'search': [],
+    'genindex': [],
+    'py-modindex': [],
+}
 
-# on_rtd is whether we are on readthedocs.org
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+# -- Options for HTMLHelp output ------------------------------------------
 
-if not on_rtd:  # only import and set the theme if we're building docs locally
-    try:
-        import sphinx_rtd_theme
-        html_theme = 'sphinx_rtd_theme'
-        html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-    except ImportError:
-        warnings.warn('WARNING: Readthedocs theme package ("sphinx_rtd_theme")'
-                      ' not found, using default theme instead.')
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'AstropyPackageTemplatedoc'
 
-# otherwise, readthedocs.org uses their theme by default, so no need to specify it
 
-# -- Resolving issue number to links in changelog -----------------------------
-github_issues_url = 'https://github.com/{0}/issues/'.format(setup_cfg['github_project'])
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {'https://docs.python.org/': None,
+                       'http://docs.astropy.org/en/stable/': None}
+intersphinx_mapping["sphinx_automodapi"] = ("https://sphinx-automodapi.readthedocs.io/en/stable/", None)  # noqa: E501, F405
 
 
-# -- Turn on nitpicky mode for sphinx (to warn about references not found) ----
-nitpicky = True
```

Comment:
You don't want to keep `nitpicky = True`?

Reply:
An oversight.  I love `nitpicky`.  Fixed.